skimage/segmentation/tests/test_random_walker.py

```diff
@@ -15,7 +15,7 @@
 PYAMG_MISSING_WARNING = r'pyamg|\A\Z'
 PYAMG_OR_SCIPY_WARNING = SCIPY_RANK_WARNING + '|' + PYAMG_MISSING_WARNING
 
-if (Version(np.__version__) >= '1.15.0'):
+if Version(np.__version__) >= '1.15.0' and Version(scipy.__version__) < '1.13':
```

Comment:
I think that the correct test is  `Version(scipy.__version__) < '1.3'`. I tested it in #3850 and CI succeeded.

Reply:
Thanks, such a brain fart there.

Reply:
I was actually about to go crazy trying to find out what was going wrong....